src/main/java/io/confluent/kafkarest/MetadataObserver.java

```diff
@@ -63,6 +63,22 @@ public MetadataObserver(KafkaRestConfig config, ZkClient zkClient) {
     return brokerIds;
   }
 
+  public Broker getBrokerById(final int brokerId) {
```

Comment:
I realized there's a better ZkUtils method for doing this -- ZkUtils.getBrokerInfo. We should do that instead since it doesn't require getting info for all the brokers.

Also, can we make this private for now since it's only used within this class? It helps to keep the public API to a minimum so we know exactly which APIs need to be maintained.